test/core/util/BUILD

```diff
@@ -113,6 +113,7 @@ grpc_cc_library(
     name = "grpc_test_util",
     srcs = TEST_UTILS_THAT_USE_GRPC_H_SOURCES + ["tls_utils.cc"],
     hdrs = TEST_UTILS_THAT_USE_GRPC_H_HEADERS + ["tls_utils.h"],
+    alwayslink = 1,
```

Comment:
why is this necessary?

Reply:
Without it now, port_isolated_runtime_environment gets stripped from the build and bazel doesn't work on rbe.

Reply:
Alright, if the MSVC builds are happy then I have no concerns.